hedera-node/test-clients/src/main/java/com/hedera/services/bdd/junit/SubProcessHapiTestNode.java

```diff
@@ -86,9 +158,16 @@ public void start() {
             final var builder = new ProcessBuilder()
                     .command(
                             javaCmd,
+                            // You can attach to any node. Node 0 at 5005, node 1 at 5006, etc. But if you need the
+                            // node to stop at startup, you can change the below line so nodeId == the node you want
+                            // to suspend at startup and the first "n" to "y".
+                            "-agentlib:jdwp=transport=dt_socket,server=y,suspend=" + (nodeId == 0 ? "n" : "n")
+                                    + ",address=*:" + (5005 + nodeId),
                             "-classpath",
                             classPath,
+                            "-Dfile.encoding=utf-8",
```

Comment:
This is basically ignored, because the node checks *native* encoding, but it cannot hurt.